azerothcore/azerothcore-wotlk
# fix(Script/BD): Fixed skill needed and more for Quest The Spectral Chalice

## Changes Proposed:
-  Added a skill check for the quest so only miners with more than 230 can pick it
-  Moved the chalice near the NPC
- Changed the gossip options for the ones on Blizzlike

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7502
- Closes https://github.com/chromiecraft/chromiecraft/issues/1504

## SOURCE:


## Tests Performed:

- .go c id 9037
- Check the new gossip text
- Check that if not a miner with more than 230 no gossip option should appear
- Check that with more than 230 gossip option shouls appear, and after clicking it the chalice should appear near him
- If someone from the group is not a miner now he should not be able to pick it


## How to Test the Changes:


- .go c id 9037
- Check the new gossip text
- Check that if not a miner with more than 230 no gossip option should appear
- Check that with more than 230 gossip option shouls appear, and after clicking it the chalice should appear near him
- If someone from the group is not a miner now he should not be able to pick it

![image](https://user-images.githubusercontent.com/87535580/130492938-6a508427-cc8f-4ab8-be08-59bd5c94d19d.png)



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
